ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -65,115 +67,145 @@ Item {
             anchors.top: titleText.bottom
             anchors.topMargin: 32
             fontPixelSize: 15
+            placeholderText: qsTr("Search by a display name or chat key")
         }
 
-        Item {
-            id: addNewContact
+        TabBar {
+            id: contactsTabBar
+            width: parent.width
             anchors.top: searchBox.bottom
-            anchors.topMargin: Style.current.bigPadding
-            width: addButton.width + usernameText.width + Style.current.padding
-            height: addButton.height
-
-
-            StatusRoundButton {
-                id: addButton
-                width: 40
-                height: 40
-                icon.name: "add"
-                type: StatusRoundButton.Type.Secondary
-                anchors.verticalCenter: parent.verticalCenter
+            anchors.topMargin: Style.current.padding
+            height: contactsBtn.height
+            background: Rectangle {
+                color: Style.current.transparent
             }
-
-            StyledText {
-                id: usernameText
-                //% "Add new contact"
-                text: qsTrId("add-new-contact")
-                color: Style.current.blue
-                anchors.left: addButton.right
-                anchors.leftMargin: Style.current.padding
-                anchors.verticalCenter: addButton.verticalCenter
-                font.pixelSize: 15
+            StatusTabButton {
+                id: contactsBtn
+                anchors.top: parent.top
+                btnText: qsTr("Contacts")
             }
-
-            MouseArea {
-                anchors.fill: parent
-                cursorShape: Qt.PointingHandCursor
-                onClicked: {
-                    addContactModal.open()
-                }
+            StatusTabButton {
+                id: pendingRequestsBtn
+                enabled: root.contactsStore.contactRequestsModel.count > 0
+                anchors.left: contactsBtn.right
+                anchors.top: parent.top
+                anchors.leftMargin: Style.current.bigPadding
+                btnText: qsTr("Pending Requests")
+            }
+            // StatusTabButton {
```

Comment:
Shall this be remove or uncommented?

Reply:
Removed. I added it because we'll need it later, but it's not ready yet